PROTO_tests/tests/client_dtypes_test.py

```diff
@@ -0,0 +1,178 @@
+import ipaddress
+import random
+
+import pytest
+
+import arkouda as ak
+from arkouda import client_dtypes
+
+INT_TYPES = [ak.int64, ak.uint64]
+IPS = [3232235777, 2222222222, 1234567890, 7]
+
+
+class TestClientDTypeTests:
```

Comment:
This name says `Test` twice and needs to be updated
```suggestion
class TestClientDTypes:
```

Reply:
Updated